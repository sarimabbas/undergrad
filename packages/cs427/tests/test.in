123 efgh 123 hello