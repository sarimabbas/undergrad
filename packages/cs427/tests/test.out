sarimabbas:tests sarimabbas$ ./a.out

Reading numbers.
123
BYTE POSITION: 3
BYTE POSITION: -1
BYTE POSITION: 8
123
BYTE POSITION: 12
BYTE POSITION: -1
BYTE POSITION: 13
BYTE POSITION: -1
BYTE POSITION: 15
BYTE POSITION: -1
BYTE POSITION: 16
BYTE POSITION: -1
BYTE POSITION: 17